src/app/components/flights-list.tsx

```diff
@@ -0,0 +1,102 @@
+"use client"
+
+import { useContext, useEffect, useState } from "react"
+import { FlightsContext } from "../contexts/flights-context"
+import { Skeleton } from "../../components/ui/skeleton";
+import FlightRow from "./flight-row";
+import { AdditionalFilters, FlightSearchInfo, SortBy, SortParameters, Stops } from "../models/flight-parameters";
+import FlightSortBy from "./sort-by";
+import Filters from "./filters";
+
+export default function FlightsList() {
+  const { isLoading, flights } = useContext(FlightsContext);
+
+  const [sortBy, setSortBy] = useState<SortParameters>({});
+  const [additionalFilters, setAdditionalFilters] = useState<AdditionalFilters>({});
+  
+  if (isLoading) {
+    return Array.from(Array(10).keys()).map((_: any, i: number) => (
```

Comment:
Do we need 10 skeleton? Instead may be we can have a single loader at the center, that would look nice.

Reply:
Not working on this for now